native_client/README.md

```diff
@@ -1,15 +1,48 @@
-# DeepSpeech native client
+# DeepSpeech native client, language bindings and custom decoder
 
-A native client for running queries on an exported DeepSpeech model.
+This folder contains a native client for running queries on an exported DeepSpeech model, bindings for Python and Node.JS for using an exported DeepSpeech model programatically, and a CTC beam search decoder implementation that scores beams using a language model, needed for training a DeepSpeech model. We provide pre-built binaries for Linux and macOS.
 
-## Requirements
+## Installation
 
-* [TensorFlow source](https://www.tensorflow.org/install/install_sources)
+To download the pre-built binaries, use `util/tc.py`:
+
+```
+python util/tc.py --target /path/to/destination/folder
+```
+
+This will download and extract `native_client.tar.xz` which includes the deepspeech binary and associated libraries as well as the custom decoder OP. `tc.py` will download binaries for the architecture of the host by default, but you can override that behavior with the `--arch` parameter. See the help info with `python util/tc.py -h` for more details.
+
+If you want the CUDA capable version of the binaries, use `--arch gpu`. Note that for now we don't publish CUDA-capable macOS binaries.
+
+If you're looking train a model, you now have a `libctc_decoder_with_kenlm.so` file that you can pass to the `--decoder_library_path` parameter of `DeepSpeech.py`.
```

Comment:
Typo...

`...you're looking train a model...`

should be

`...you're looking to train a model...`